stripe/src/test/java/com/stripe/android/util/StripeNetworkUtilsTest.java

```diff
@@ -0,0 +1,91 @@
+package com.stripe.android.util;
+
+import android.support.annotation.NonNull;
+
+import com.stripe.android.model.Card;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Test class for {@link StripeNetworkUtils}
+ */
+@RunWith(RobolectricTestRunner.class)
+@Config(sdk = 23)
+public class StripeNetworkUtilsTest {
+
+    private static final String CARD_NUMBER = "4242424242424242";
+    private static final String CARD_CVC = "123";
+    private static final String CARD_CITY = "San Francisco";
+    private static final String CARD_CURRENCY = "USD";
+    private static final String CARD_ADDRESS_L1 = "123 Main Street";
+    private static final String CARD_ADDRESS_L2 = "906";
+    private static final String CARD_ZIP = "94107";
+    private static final String CARD_STATE = "CA";
+    private static final String CARD_COUNTRY = "US";
+    private static final String CARD_NAME = "J Q Public";
```

Comment:
Same comment as another PR: I have a slight preference for alphabetizing this type of stuff, but not a big deal.